app/javascript/dashboard/routes/dashboard/Dashboard.vue

```diff
@@ -77,14 +79,51 @@ export default {
       const { show_secondary_sidebar: showSecondarySidebar } = this.uiSettings;
       return showSecondarySidebar;
     },
+    previouslyUsedDisplayType() {
+      const {
+        previously_used_conversation_display_type: conversationDisplayType,
+      } = this.uiSettings;
+      return conversationDisplayType;
+    },
+  },
+  watch: {
+    isDesktopView() {
+      const { LAYOUT_TYPES } = wootConstants;
+      this.updateUISettings({
+        conversation_display_type: !this.isDesktopView
+          ? LAYOUT_TYPES.EXPANDED
+          : this.previouslyUsedDisplayType,
+      });
+    },
   },
   mounted() {
+    this.handleResize();
+    let throttled = false;
+    const delay = 150;
+    window.addEventListener('resize', () => {
+      if (!throttled) {
+        this.handleResize();
+        throttled = true;
+        setTimeout(() => {
+          throttled = false;
+        }, delay);
+      }
+    });
     bus.$on(BUS_EVENTS.TOGGLE_SIDEMENU, this.toggleSidebar);
   },
   beforeDestroy() {
+    window.removeEventListener('resize', this.handleResize);
```

Comment:
This won't remove the listener, since the function signature won't match with the one given originally, we can add the throttle in the handle resize itself

Another option is to write a throttle handler, it will return a throttled function

```js
function throttle(fn, delay) {
  let throttled = false;
  return function (...args) {
    if (throttled) {
      return;
    }
    throttled = true

    setTimeout(() => {
      throttled = false;
      fn(...args);
    }, delay);
  };
}
```

But for now, maybe we can just add the throttle to the `handleResize` itself. Upto you, on what seems better

Reply:
I pushed the changes by this commit https://github.com/chatwoot/chatwoot/pull/6134/commits/a7664eaa0959e89673f92f67b7930c8bac334920 Can you check it?